userspace/src/usbip/usbip.c

```diff
@@ -50,47 +50,53 @@ struct command {
 
 static const struct command cmds[] = {
 	{
-		.name  = "help",
```

Comment:
I do not known why are these changes here. I created my branch from updatream master.

Reply:
I noticed now. It's Visual Studio feature. I will repair.